inventory/staging.yml

```diff
@@ -18,6 +18,7 @@ all:
 
     dokkus:
       hosts:
+        dokku9.code4sa.org:
```

Comment:
i think it's best not to add dokku9 here since dokku wasn't installed using ansible on this machine

but I think we can get away with using it like a dokku server further down.